site/app/templates/calendar/Calendar.twig

```diff
@@ -0,0 +1,70 @@
+{% import _self as self %}
+
+<script>
+    let gradeables_by_date = {{ gradeables_by_date|json_encode|raw }};
+    let curr_year = {{ curr_year }};
+    let curr_month = {{ curr_month }};
+    let curr_day = {{ curr_day }};
+
+    window.onload = function () {
+        loadCalendar({{ view_month }}, {{ view_year }});
+    }
+</script>
+
+{# Begin Calendar #}
+<div class="content">
+    <h1>Calendar</h1>
+    <div class="cal-content">
+        <div class="cal-toolbar">
+            {# Replace by js function call #}
+        </div>
+
+        <div class="calendar" style='margin-bottom: 20px' id="full_calendar">
+        </div>
+    </div>
+
+    {# List items in table #}
+    <div class="lst-content">
+        {% if show_only_calendar == 0 %}
+            {% if gradeables_by_section|length > 0 %}
+                <div id="gradeables">
+                {% for section_num, info in gradeables_by_section %}
+                    <div style='margin-bottom: 20px'>
+                        <div id="{{ info.section_id }}" class="course-section-heading">
+                            <div class="course-section-title">
+                                {{ info.title }}
+                                {% if info.subtitle != "" %}
+                                    &nbsp;&nbsp;<em>{{ info.subtitle }}</em>
+                                {% endif %}
+                            </div>
+                        </div>
+
+                        <table class='table table-striped table-bordered persist-area' style='text-align: left'>
```

Comment:
Lets make this page mobile-friendly from the start, the long list of gradeable dates after the calendar should be easy. For now, we'll just make sure the calendar fits on the screen for a mobile device and we can optimize it for useability later, I imagine it will need special functionality for mobile vs tablet.

Add the class `mobile-table` to this table and give it an id, something like ` id='calendar-list'` or whatever you prefer.